coordinator/src/main/scala/filodb.coordinator/queryplanner/LogicalPlanUtils.scala

```diff
@@ -66,7 +66,7 @@ object LogicalPlanUtils extends StrictLogging {
       case lp: LabelValues                 => TimeRange(lp.startMs, lp.endMs)
       case lp: LabelCardinality            => TimeRange(lp.startMs, lp.endMs)
       case lp: LabelNames                  => TimeRange(lp.startMs, lp.endMs)
-      case lp: TsCardinalities             => throw new IllegalArgumentException("no time params for TsCardinalitites")
+      case lp: TsCardinalities             => TimeRange(0, Long.MaxValue)  // TODO(a_theimer)
```

Comment:
HAP `materialize` always uses this function, so I need TsCardinalities to map to some `TimeRange` here. Is this sufficient?

Reply:
no longer applicable